### DISCUSSION TITLE
How to create a library for a single platform?

### DISCUSSION THREAD

**Discussion by dustinbahr** [callstack/react-native-builder-bob#457] [Q&A]
Is there a way to use `create-react-native-library` to setup a library for a single target? For example iOS only, or Android only.

Alternatively, is there a guide on how to remove either iOS or Android from the boilerplate that is created?

**Comment by satya164** [comment]
You can just delete the iOS or android folders.

**Comment by dustinbahr** [comment]
Is this the only, and/or, best way? It seems there would still be other unnecessary references to the `ios` or `android` folders still in the project. 

A few examples:
```
- package.json
- turbo.json
- ci.yml
...
```
